### ISSUE TITLE
[Supporting the platform-Social Embedded-Sign In]:"Powered by Microsoft Embedded Social" text is not adopting system in high contrast setting.

### ISSUE THREAD

**Issue by billycamp** [Microsoft/EmbeddedSocial-Android-SDK#26]
Priority: Low

Repro Steps
Pre-requisite:
Apply High Contrast : (Settings> Accessibility> High contrast>Toggle on)

1. Install ""One Bus Away"" app on Android.  
2. Start ""One Bus Away"" app from Android phone.
3. ""One Bus Away "" Home page will get appear.
4. Navigate to ""Open Navigation Drawer"" button and press ENTER (or double tap) to open.
5. Navigate to ""Sign In"" and press ENTER (or double tap) to open. 
6. “Sign in” page will get appear.
7. Verify if  ""Powered by Microsoft Embedded Social"" text is adopting system control in all High Contrast Setting.

Expected
""Powered by Microsoft Embedded Social"" text should be adopt system controls in high contrast setting.

Actual
""Powered by Microsoft Embedded Social"" text is not adopting system control in high contrast setting.

Note: 
1> Issue is repro throughout the application for ""Social Embedded"" part.

MAS Rule: MAS4.3.1

Impact
People with vision impairment and cognitive disabilities will get confused if text is not adopting system controls in High Contrast setting.

Test Environment
OS: Android
Device Model No. : Nexus 5
Android Version : 6.0.1
App: Social Embedded
App Version : v2.2.3 (84)
Tool : High Contrast Setting
